internal/cmd/pipeline/command_create.go

```diff
@@ -23,15 +22,20 @@ func (c *command) newCreateCommand() *cobra.Command {
 		RunE:  c.create,
 		Example: examples.BuildExampleString(
 			examples.Example{
-				Text: "Create a new Stream Designer pipeline",
-				Code: `confluent pipeline create --name test-pipeline --ksql-cluster lksqlc-12345 --description "this is a test pipeline"`,
+				Text: `Create a Stream Designer pipeline with description "my-pipeline".`,
```

Comment:
```suggestion
				Text: `Create a Stream Designer pipeline with description "example pipeline".`,
```

Reply:
Since `"example pipeline"` is the argument for `--description` in this example while `my-name` is the argument for `--name`.

Also there's still a mismatch between this text and the golden file, we can merge as soon as it's resolved!

Reply:
Oops, my bad. My earlier suggestion was wrong.
```suggestion
				Text: `Create a Stream Designer pipeline with name "my-pipeline".`,
```